flocker/node/agents/ebs.py

```diff
@@ -306,13 +330,9 @@ def _enable_boto_logging():
     """
     Make boto log activity using Eliot.
     """
-    logger = logging.getLogger("boto")
-    logger.addHandler(EliotLogHandler())
-
-    # It seems as though basically all boto log messages are at the same
-    # level.  Either we can see all of them or we can see none of them.
-    # We'll do some extra filtering in the handler.
+    logger = logging.getLogger("boto3")
     logger.setLevel(logging.DEBUG)
```

Comment:
Given we're only logging `INFO` messages, presumably we don't need to set level to DEBUG but rather to INFO?